.github/workflows/deploy-staging-and-prepare-release.yaml

```diff
@@ -93,8 +129,31 @@ jobs:
           DEPLOYMENT_BUCKET: ${{ secrets.AWS_CLOUD_STAGING_DEPLOYMENT_BUCKET }}
           CDN_DISTRIBUTIONS: ${{ secrets.AWS_CLOUD_STAGING_CDN_DISTRIBUTIONS }}
 
+      - name: Update check
+        if: always() && ${{ steps.create-check.outputs.CHECK_ID }}
+        uses: ./.github/actions/checks/update-pull-request-check
+        with:
+          CHECK_ID: ${{ steps.create-check.outputs.CHECK_ID }}
+          CHECK_NAME: "Deploy_staging"
+          STATUS: "${{ steps.deploy.outcome }}"
+          CONCLUSION: "${{ steps.deploy.conclusion }}"
+          TITLE: "Deploy staging"
+          SUMMARY: "Deploy finished with status ${{ steps.deploy.outcome }}. To check details: ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}"
+          DETAILS_URL: "${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}"
+
+      - name: Notify Slack
+        if: ${{ always() }}
+        env:
+          JOB_DEPLOYMENT_KIND: staging
+          JOB_STATUS: ${{ job.status }}
+          SLACK_WEBHOOK_URL: ${{ secrets.SLACK_CLOUD_DEPLOYMENTS_WEBHOOK_URL }}
+          SLACK_MENTION_GROUP_ID: ${{ secrets.SLACK_DASHBOARD_GROUP_ID }}
+          JOB_TITLE: "Dashboard deployment of ${{ env.CUSTOM_VERSION || env.VERSION }} to ${{ env.ENVIRONMENT }}"
+        run: |
+          python3 ./.github/workflows/notify/notify-slack.py
```

Comment:
https://github.com/slackapi/slack-github-action why not?

Reply:
We already use it like this in a few other workflows, I think if we want to change it it should be everywhere and in separate PR